core/src/main/java/io/questdb/std/Files.java

```diff
@@ -37,6 +37,8 @@ public final class Files {
 
     public static final int DT_DIR = 4;
     public static final int DT_FILE = 8;
+    public static final int DT_LNK = 10; // soft link
```

Comment:
@marregui could you include a test and a fix for #2884 into this PR? I've started working in that issue, but recalled that you're working on symlink support for partitions.

Reply:
I have created test `testServerMainCreateTableMoveItsFolderAwayAndSoftLinkIt` to exemplify the use, and now the cursor is symlink aware.